vm/cuttlefish/cuttlefish.go

```diff
@@ -81,6 +81,18 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		return nil, fmt.Errorf("failed to get root access to device: %s", err)
 	}
 
+	if err := inst.runOnHost(5*time.Minute, "adb shell setprop persist.dbg.keep_debugfs_mounted 1"); err != nil {
+		return nil, fmt.Errorf("failed to set setprop persist.dbg.keep_debugfs_mounted 1: %s", err)
+	}
+
+	if err := inst.runOnHost(5*time.Minute, "adb shell mount -t debugfs debugfs /sys/kernel/debug"); err != nil {
+		return nil, fmt.Errorf("failed to mount debugfs to /sys/kernel/debug: %s", err)
+	}
+
+	if err := inst.runOnHost(5*time.Minute, "adb shell chmod 0755 /sys/kernel/debug"); err != nil {
```

Comment:
May be faster to do 3 commands in 1 shell command rather than establish 3 separate ssh and adb connections. Also fewer lines and easier to extend in future.

Reply:
That's a good point. I'll update it.